sdk/appconfiguration/app-configuration/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/app-configuration",
   "author": "Microsoft Corporation",
   "description": "An isomorphic client library for the Azure App Configuration service.",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Why didnt the automation that increments version number do this?

Reply:
I couldn't find it yesterday.

Reply:
I was filtering with App Configuration label and it turned out there was no label on the automated PR https://github.com/Azure/azure-sdk-for-js/pull/16269.

Asked @praveenkuttappan  in the teams' channel